src/Components/MovieForm/LabeledInput/LabeledInput.jsx

```diff
@@ -1,12 +1,20 @@
 import styles from './LabeledInput.module.css';
 
-export const LabeledInput = ({ label, value, onChange, id, type = 'text' }) => {
+export const LabeledInput = ({ label, value, onChange, id, handleBlur, type = 'text' }) => {
```

Comment:
Props are usually called "onSomething":

```suggestion
export const LabeledInput = ({ label, value, onChange, id, onBlur, type = 'text' }) => {
```